### PR TITLE
Add rearm and refuel documentation

### PR DESCRIPTION
**Pull request by GitHawk**
- [x] Add feature description
- [x] Add developer description
- [x] Add mission designer description

### CODE REVIEW

**Comment by GitHawk** [comment]
@bux578 Can you give me feedback on my additions?

File: wiki/feature/rearm.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+| Caliber in mm            |   6 |   7 |   8 |  13 | 19 | 20 | 25 | 30 | 35 | 39 | 40 | 60 | 70 | 80 | 82 | 100 | 105 | 120 | 122 | 125 | 155 | 230 | 250 |
+|--------------------------|-----|-----|-----|-----|----|----|----|----|----|----|----|----|----|----|----|-----|-----|-----|-----|-----|-----|-----|-----|
+| Number of rounds per box | 500 | 500 | 400 | 100 | 50 | 50 | 40 | 25 | 34 | 24 | 10 |  2 |  2 |  2 |  2 |   1 |   1 |   1 |   1 |   1 |   1 |   1 |   1 |
+
+The caliber of the weapon will be rounded to the nearest number and the number of rounds is calculated using the table above.
+To fully rearm 2000 rounds of 7.62mm machine gun, you need five ammo boxes.
+
+## 3. Limiting the ammunition supply
+With ACE 3.6.0 ammunition supply can be limited on ammo trucks.
```
Comment (bux578) [inline wiki/feature/rearm.md]: ACE3

Comment (bux578) [inline wiki/feature/rearm.md]: ACE3

File: wiki/feature/rearm.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+### 3.1 Rearming with setting `Unlimited ammo supply`
+All ammo trucks have an unlimited ammunition supply by default. You can rearm as many vehicles as often as you like.
+
+### 3.2 Rearming with setting `Limited ammo supply based on caliber`
+On this setting all ammo trucks have a limited ammunition supply. The supply consumption is based on "caliber points". With 1000 points worth of ammunition supply, you can rearm four magazines of 250mm caliber (i.e. bombs) or eight magazines of 120mm caliber (i.e. tank rounds) or 125 magazines of 7.62mm caliber.
+
+### 3.3 Rearming with setting `Only specific Magazines`
+On this setting, the mission creator mission creator has to use the framework to add specific magazines to the ammo truck(s). This allows you to create munition specific ammo trucks i.e. one ammo truck for missiles and one for bombs.
```
Comment (bux578) [inline wiki/feature/rearm.md]: mission creator mission creator

File: wiki/feature/refuel.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+### The nozzle was dropped. What's wrong?
+You can only use the nozzle within a close distance from its source. You may need to park closer.
+
+### The engine of the fuel truck is broken. Why?
+While refueling, you can't move the fuel truck. Return the nozzle.
+
+### How do I replenish the fuel supply on a fuel truck?
+Please check the framework description for more details.
+
+### Something broke, I can't use the fuel truck / nozzle any longer. What to do?
```
Comment (bux578) [inline wiki/feature/refuel.md]: puh, this might be too coding specific for a FAQ, dunno

Comment (GitHawk) [inline wiki/feature/refuel.md]: So just write "There is a way to reset the fuel truck. Please check the framework description for more details"?

File: wiki/feature/rearm.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+All ammo trucks have an unlimited ammunition supply by default. You can rearm as many vehicles as often as you like.
+
+### 3.2 Rearming with setting `Limited ammo supply based on caliber`
+On this setting all ammo trucks have a limited ammunition supply. The supply consumption is based on "caliber points". With 1000 points worth of ammunition supply, you can rearm four magazines of 250mm caliber (i.e. bombs) or eight magazines of 120mm caliber (i.e. tank rounds) or 125 magazines of 7.62mm caliber.
+
+### 3.3 Rearming with setting `Only specific Magazines`
+On this setting, the mission creator has to use the framework to add specific magazines to the ammo truck(s). This allows you to create munition specific ammo trucks i.e. one ammo truck for missiles and one for bombs.
+
+### 3.4 Checking the ammunition supply
+- Interact with the fuel truck <kbd>⊞&nbsp;Win</kbd> (ACE3 default key bind `Interact Key`).
```
Comment (sargken) [inline wiki/feature/rearm.md]: Ammo truck? Typo?

Comment (GitHawk) [inline wiki/feature/rearm.md]: Yes, copy paste error.

**Pull request merged**